poetry.lock

```diff
@@ -512,6 +548,20 @@ category = "dev"
 optional = false
 python-versions = "*"
 
+[[package]]
+name = "modelldcatnotordf"
+version = "0.0.1"
+description = "A library for mapping a data catalog to rdf"
```

Comment:
Kanskje litt misvisende description. Data catalog er Stig sitt bibliotek men mitt er modelldcatno

Reply:
Det er nedlastet informasjon fra ditt bibliotek :P

Reply:
He, he, da skal jeg oppdatere der. :grin:

Reply:
Men ikke så viktig, bare kjør på for min del